support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -146,7 +146,7 @@ function getIsRemoteFromAcquisitionData(): boolean {
 
   try {
     const data = JSON.parse(queryString);
-    return data ? data.isRemote === 'true' : false;
+    return data ? data.isRemote === true : false;
```

Comment:
Could this just be:

```js
return data ? data.isRemote : false;
```

Reply:
Yes, although I might prefer `return data ? !!data.isRemote : false` . I hesitated to do this as prefer being explicit, but this is safer, particularly given unclear how to test locally.